manifests/deis-workflow-rc.yml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: deis-workflow
-          image: quay.io/deisci/workflow:v2-beta
+          image: quay.io/deis/workflow:v2-beta
```

Comment:
And we're using the `deisci` org for *everything* until Deis v2 has an actual release. There shouldn't ever be test images deployed to the `deis` org on Docker Hub or quay.io.